### DISCUSSION TITLE
Support for Claude 3.5 ?

### DISCUSSION THREAD

**Discussion by erik-ashby-helpshift** [danielmiessler/Fabric#642] [Q&A]
So I just signed up for Claude API since they have released 3.5, and wanted to try it with Fabric;  However after adding the API key , I can only see the following models/    I do not see claude-3-5-sonnet.  Is this not yet supported?

Claude Models:
claude-3-opus-20240229
claude-3-sonnet-20240229
claude-3-haiku-20240307
claude-2.1

**Comment by ksylvan** [comment]
From the latest pull, you see this. git pull and upgrade to the latest.
```
Claude Models:
claude-3-5-sonnet-20240620
claude-3-opus-20240229
claude-3-sonnet-20240229
claude-3-haiku-20240307
claude-2.1
```

**Reply by erik-ashby-helpshift** [reply]
erikashby@Ash-Omen:~/fabric$ git pull https://github.com/danielmiessler/fabric.git
From https://github.com/danielmiessler/fabric
 * branch            HEAD       -> FETCH_HEAD
Already up to date.

erikashby@Ash-Omen:~/fabric$ fabric -u
Updating patterns...
Downloaded zip file successfully.
Extracted zip file successfully.
Patterns updated successfully.

(Restart)
erikashby@Ash-Omen:~/fabric$ fabric --listmodels
Claude Models:
claude-3-opus-20240229
claude-3-sonnet-20240229
claude-3-haiku-20240307
claude-2.1

>> I must be missing something/

**Reply by erik-ashby-helpshift** [reply]
@ksylvan , any ideas?
